src/session.ts

```diff
@@ -894,7 +896,8 @@ Type 'help' to get help.
             action: async (): Promise<void> => {
                 await vscode.env.openExternal(
                     vscode.Uri.parse("https://dotnet.microsoft.com/en-us/download/dotnet-framework"));
-            }}]
+            }
+        }]
```

Comment:
Interesting...saving the file re-ran the formatter (which has had dependent packages updated...) so it looks like it now wants to format differently.